boards/arm/udoo_neo_full_m4/doc/udoo_neo_full.rst

```diff
@@ -0,0 +1,216 @@
+  - 22x GPIO (M4)
+
+- Other
+
+  - MicroSD card slot (8-bit SDIO interface)
+  - Power status LED (green)
+  - 2x user LED (red and orange)
+
+- Power
+
+  - 5 V DC Micro USB
+  - 6-15 V DC jack
+  - RTC battery connector
+
+- Debug
+
+  - pads for soldering of JTAG 14-pin connector
+
+- Sensor
+
+  - 3-Axis Accelerometer
+  - 3-Axis Magnetometer
+  - 3-Axis Digital Gyroscope
+  - 1x Sensor Snap-In I2C connector
+
+- Expansion port
+
+  - Arduino interface
+
+For more information about the MCIMX6X SoC and UDOO Neo Full board,
+see these references:
+
+- `NXP i.MX 6SoloX Website`_
+- `NXP i.MX 6SoloX Datasheet`_
+- `NXP i.MX 6SoloX Reference Manual`_
+- `UDOO Neo Website`_
+- `UDOO Neo Getting Started`_
+- `UDOO Neo Documentation`_
+- `UDOO Neo Datasheet`_
+- `UDOO Neo Schematics`_
+
+Supported Features
+==================
+
+The UDOO Neo Full board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | general purpose input/output        |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/udoo_neo_full_m4/udoo_neo_full_m4_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+The UDOO Neo Full board was tested with the following pinmux
+controller configuration.
+
++---------------+-----------------+---------------------------+
+| Board Name    | SoC Name        | Usage                     |
++===============+=================+===========================+
+| J4 RX         | UART5_RX_DATA   | UART Console              |
++---------------+-----------------+---------------------------+
+| J4 TX         | UART5_TX_DATA   | UART Console              |
++---------------+-----------------+---------------------------+
+
+System Clock
+============
+
+The MCIMX6X SoC is configured to use the 24 MHz external oscillator
+on the board with the on-chip PLL to generate core clock.
+PLL settings for M4 core are set via code running on A9 core.
+
+Serial Port
+===========
+
+The MCIMX6X SoC has six UARTs. UART5 is configured for M4 console and the
+remaining are used by A9 core or not used.
+
+Programming and Debugging
+*************************
+
+The M4 core does not have flash memory and is not provided a clock
```

Comment:
Can we add details like in colibri_imx7d_m4 doc's have for loading via u-boot?

Reply:
Agreed

Reply:
I haven't used u-boot. Only loaded it from Linux user space application which is part of the particular Linux distribution used (UDOObuntu). Anyway, I will add some instructions for this distribution at least (and the link to program, which is distribution independent).

Reply:
Added details about the loading of second core image.